### PR TITLE
[4192][12.0][IMP] connector_ebisumart: aware return transfer scheduled_date

### PR DESCRIPTION
**Pull request by AungKoKoLin1997**
[4192](https://www.quartile.co/web#id=4192&cids=3&menu_id=505&action=1457&model=project.task&view_type=form)

### CODE REVIEW

File: connector_ebisumart/views/sale_order_views.xml [hunk 1]
```diff
@@ -7,6 +7,7 @@
         <field name="arch" type="xml">
             <field name="confirmation_date" position="after">
                 <field name="ebisumart_send_date" readonly="1" />
+                <field name="ebisumart_cancel_date" readonly="1" />
```
Comment (kanda999) [inline connector_ebisumart/views/sale_order_views.xml]: Display only when `cancel_in_ebisumart` is `True`

**Review by kanda999 — APPROVED** [review]
It works well as expected.
Please run pre-commit

**Comment by AungKoKoLin1997** [comment]
> Please run pre-commit

@kanda999 I run. Why I need to run again?

**Comment by kanda999** [comment]
@AungKoKoLin1997 
I thought line 11 in `sale_order_views.xml` was too long. If you already pre-commit, no problem.

**Pull request merged**